module/Database/src/Database/Service/Member.php

```diff
@@ -129,6 +242,28 @@ public function getMember($id)
         }
     }
 
+    /**
+     * Get temporary member info.
+     *
+     * @param int $id
+     *
+     * @return MemberTempModel
+     */
+    public function getMemberTemp($id)
+    {
+        try {
+            return array(
+                'member' => $this->getMemberTempMapper()->find($id),
+                'simple' => false
+            );
+        } catch (\Doctrine\ORM\NoResultException $e) {
+            return array(
+                'member' => $this->getMemberTempMapper()->findSimple($id),
+                'simple' => true
+            );
+        }
```

Comment:
Is deze oplossing nodig? Want temp members zullen altijd 'simple' zijn. Geen kans dat die aan organen enzo vast zitten.